doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -118,10 +119,12 @@ Other options:
 * :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_APP`
 * :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_STATIC`
 * :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_MAX_LEN`
+* :kconfig:option:`CONFIG_AWS_IOT_BROKER_HOST_NAME_MAX_LEN`
 
 
 .. note::
    If you are using a longer device ID that is either set by the option :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_STATIC` or passed in during initialization, it might be required to increase the value of the option :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_MAX_LEN` for proper initialization of the library.
+   Similarly, if your AWS host name set by the option :kconfig:option:`CONFIG_AWS_IOT_BROKER_HOST_NAME` or passed in during initialization is longer than the default value of :kconfig:option:`CONFIG_AWS_IOT_BROKER_HOST_NAME_MAX_LEN`, it is required to increase this value for proper initialization of the library.
```

Comment:
Do you think it is better to drop line 128? This note block might become rather large and the information given in this sentence was already provided at line 92, 93 and 94.

I also noticed a section at line 156 which talks about setting the client-ID at runtime. Maybe it would be better to revert all current changes and just change this section to incorporate setting the host name runtime or add an additional section with this information? What do you think?

Reply:
Lets stick to AWS hostname for now